worker-api/src/main/java/com/microfocus/caf/api/worker/QueueTaskMessage.java

```diff
@@ -0,0 +1,265 @@
+/*
+ * Copyright 2015-2021 Micro Focus or one of its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.microfocus.caf.api.worker;
```

Comment:
Don't start a new package.  Put this in `com.hpe.caf.api.worker` like the rest.  I would like to get them moved but that's not part of this Jira and this will just leave it inconsistent.